### PR TITLE
[4.0] Crowdin comments

### PR DESCRIPTION
**Pull request by brianteeman**
This PR updates the comments in language strings to a format that crowdin understands as requested by @conconnl https://github.com/joomla/core-translations/issues/1

Basically comments are moved to the end of the line in the *.ini

Note comments which are not for translators such as a comment used to indicate a group of strings have not been changed

### CODE REVIEW

**Comment by conconnl** [comment]
@HLeithner & @zero-24 Please look at it and merge.

File: administrator/language/en-GB/joomla.ini [hunk 1]
```diff
@@ -6,8 +6,7 @@
 ; Common boolean values
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
-; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s" ; Keep this string on top
```
Comment (HLeithner) [inline administrator/language/en-GB/joomla.ini]: Don't think that makes sense for crowdin, it's more important that it's above the string  for developer

File: administrator/language/en-GB/lib_joomla.ini [hunk 2]
```diff
@@ -6,8 +6,7 @@
 ; Common boolean values
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
-; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s" ; Keep this string on top
```
Comment (HLeithner) [inline administrator/language/en-GB/lib_joomla.ini]: Don't think that makes sense for crowdin, it's more important that it's above the string  for developer

File: api/language/en-GB/joomla.ini [hunk 3]
```diff
@@ -6,8 +6,7 @@
 ; Common boolean values
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
-; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s" ; Keep this string on top
```
Comment (HLeithner) [inline api/language/en-GB/joomla.ini]: Don't think that makes sense for crowdin, it's more important that it's above the string  for developer

File: language/en-GB/joomla.ini [hunk 4]
```diff
@@ -6,8 +6,7 @@
 ; Common boolean values
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
-; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s" ; Keep this string on top
```
Comment (HLeithner) [inline language/en-GB/joomla.ini]: Don't think that makes sense for crowdin, it's more important that it's above the string  for developer

File: language/en-GB/lib_joomla.ini [hunk 5]
```diff
@@ -6,8 +6,7 @@
 ; Common boolean values
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
-; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s" ; Keep this string on top
```
Comment (HLeithner) [inline language/en-GB/lib_joomla.ini]: Don't think that makes sense for crowdin, it's more important that it's above the string  for developer

**Review by HLeithner — CHANGES_REQUESTED** [review]
Rest looks good

**Comment by brianteeman** [comment]
Thanks for checking @HLeithner - perfectly correct and updated

File: administrator/language/en-GB/lib_joomla.ini [hunk 6]
```diff
@@ -7,7 +7,7 @@
 ; Note: YES, NO, TRUE, FALSE are reserved words in INI format.
 
 ; Keep this string on top
-JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
+ JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
```
Comment (HLeithner) [inline administrator/language/en-GB/lib_joomla.ini]:
```suggestion
JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
```

**Review by HLeithner — CHANGES_REQUESTED** [review]

**Comment by Bakual** [comment]
I think we looked into this already years ago and I know it didn't work at that time. I don't exactly remember the reasons however.
One was certainly that com_localise doesn't recognise those comments, but I think at that time Joomla couldn't handle them (language files broke I think). So please make sure everything related to language handling works when having the comments at the end (eg debugging languages)

**Comment by brianteeman** [comment]
I'm just doing the work that was requested - no need for the thumbs

**Comment by brianteeman** [comment]
> So please make sure everything related to language handling works when having the comments at the end (eg debugging languages)

Already tested before submitting a pr

**Review by HLeithner — APPROVED** [review]

File: language/en-GB/mod_footer.ini [hunk 7]
```diff
@@ -2,9 +2,8 @@
 ; (C) 2005 Open Source Matters, Inc. <https://www.joomla.org>
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
-; Note : %date% will be auto replaced by current year !Don't translate
 
 MOD_FOOTER="Footer"
-MOD_FOOTER_LINE1="Copyright &#169; %date% %sitename%. All Rights Reserved."
+MOD_FOOTER_LINE1="Copyright &#169; %date% %sitename%. All Rights Reserved." ; Note : %date% will be auto replaced by current year !Don't translate
```
Comment (richard67) [inline language/en-GB/mod_footer.ini:7]: Doesn't the same apply also on `%sitename%`?

Comment (brianteeman) [reply language/en-GB/mod_footer.ini:7]: Probably - but beyond the scope of this PR which is just about changing where the existing comments are

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by obuisard** [comment]
I have tested this item :white_check_mark: successfully on e56f3e427621926949fdc2d7ea2a1e99a16eb151<br /><br />The changes have no incidence in the outputs (like, for instance, creating an instance of the footer module)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34671">issues.joomla.org/tracker/joomla-cms/34671</a>.</sub>

**Comment by brianteeman** [comment]
Conflicts resolved again.

Please either merge this PR **before the freeze** that was requested by the language co-ordinators or close it. Not wasting my time any more

**Comment by HLeithner** [comment]
If tools around this PR breaks, we have to fix them or retire them if no longer used. Crowdin is our current translation platform and we should support it as good as we can.

**Comment by brianteeman** [comment]
Thanks @HLeithner 

over to you now @conconnl

**Pull request merged**